tests/phpunit/payment-methods/class-wc-stripe-upe-payment-gateway-test.php

```
@@ -4412,4 +4412,382 @@ public function test_add_email_currency_conversion_notice_outputs_plain_text_for
+		array $account_data,
+		bool $short_descriptor_on,
+		bool $expect_full,
+		bool $expect_suffix,
+		string $expected_value
+	) {
+		$order       = WC_Helper_Order::create_order();
+		$order_id    = $order->get_id();
+		$customer_id = 'cus_mock';
+
+		// Configure the statement descriptor settings.
+		$stripe_settings                         = WC_Stripe_Helper::get_stripe_settings();
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Helper' in method 'test_statement_descriptor_for_payment_types'. (undefined)

(StaticAccess)